packages/layout/src/Box/Box.constants.ts

```diff
@@ -1,62 +1,166 @@
-export const VARIANT = {
-  ghost: "bg-transparent",
-};
-
 export const BOX_PROPS = {
+  h: {
+    rule: "height",
+    type: "spacing",
+  },
+  w: {
+    rule: "height",
+    type: "spacing",
+  },
   p: {
     rule: "padding",
     type: "spacing",
+    direction: "all",
   },
   m: {
     rule: "margin",
     type: "spacing",
+    direction: "all",
   },
   pt: {
     rule: "padding",
-    type: "spacing-top",
+    type: "spacing",
+    direction: "top",
   },
   pl: {
     rule: "padding",
-    type: "spacing-left",
+    type: "spacing",
+    direction: "left",
   },
   pb: {
     rule: "padding",
-    type: "spacing-bottom",
+    type: "spacing",
+    direction: "bottom",
   },
   pr: {
     rule: "padding",
-    type: "spacing-right",
+    type: "spacing",
+    direction: "right",
   },
   px: {
     rule: "padding",
-    type: "spacing-x",
+    type: "spacing",
+    direction: "x",
   },
   py: {
     rule: "padding",
-    type: "spacing-y",
+    type: "spacing",
+    direction: "y",
   },
   mt: {
     rule: "margin",
-    type: "spacing-top",
+    type: "spacing",
+    direction: "top",
   },
   ml: {
     rule: "margin",
-    type: "spacing-left",
+    type: "spacing",
+    direction: "left",
   },
   mb: {
     rule: "margin",
-    type: "spacing-bottom",
+    type: "spacing",
+    direction: "bottom",
   },
   mr: {
     rule: "margin",
-    type: "spacing-right",
+    type: "spacing",
+    direction: "right",
   },
   mx: {
     rule: "margin-inline",
-    type: "spacing-x",
+    type: "spacing",
+    direction: "x",
   },
   my: {
     rule: "margin-block",
-    type: "spacing-y",
+    type: "spacing",
+    direction: "y",
+  },
+  bg: {
+    rule: "background-color",
+    type: "color",
+  },
+  r: {
```

Comment:
What're your thoughts on using shorthand properties like `r` vs. more verbose ones like `rounded` (same for `bd` vs. `border`)? 

I know we use `m` / `p` which are common enough that they're easy to understand but `r` and `bd` are a bit harder to parse, especially when you start adding modifiers like direction and additional properties (e.g. `bdc`). 

I'm not strongly opposed to the shorter properties as they're nicer to write, but I'm also worried it'll make components harder to read and the component api harder to learn/remember.

Reply:
Yeah I was going back and forth myself on it. I think between your thought and sleeping on it verbosity makes more sense for these. Easy swap 👍